docs/stdlib/string.rst

```diff
@@ -259,9 +259,9 @@ Strings
 
     Case-sensitive simple string matching.
 
-    Returns ``true`` if the *value* ``V`` matches the *pattern* ``P``
-    and ``false`` otherwise.  The operator ``not like`` is
-    the negation of ``like``.
+    Returns ``true`` if the *value* (the ``str`` on the left) matches the
+    *pattern* (the ``str`` on the right) and ``false`` otherwise. The operator
+    ``not like`` is the negation of ``like``.
```

Comment:
I wasn’t sure what the `V` and `P` were referring to, so I thought something like this might be more clear. But let me know if you think something else would be better, or if you prefer how it originally was.

Reply:
Looks good to me.